libftl/ftl_private.h

```diff
@@ -266,6 +306,14 @@ typedef struct _ftl_ingest_t {
   struct _ftl_ingest_t *next;
 }ftl_ingest_t;
 
+typedef struct
+{
+    ftl_handle_t* handle;
+    BOOL(*change_bitrate_callback)(void*, uint64_t);
+    void* context;
+    uint64_t ullInitialEncodingBitrate;

```

Comment:
Actually I will remove all Hungarian notation to keep it consistent with rest of ftl. Added TODO item.